app/components/navbar.tsx

```diff
@@ -235,6 +239,78 @@ function MobileMenu() {
   )
 }
 
+// Timing durations used to control the speed of the team ring in the profile button.
+// Time is seconds per full rotation
+const durations = {
+  initial: 20,
+  hover: 10,
+  focus: 4,
+  active: 0.25,
+}
+
+function ProfileButton({
+  imageUrl,
+  imageAlt,
+  team,
+  user,
+  hasActiveMagicLink,
+}: {
+  imageUrl: string
+  imageAlt: string
+  team: OptionalTeam
+  user: User | null
+  hasActiveMagicLink: boolean
+}) {
+  const controls = useAnimation()
+  const [ref, state] = useElementState()
+  const isMascotAvatar = alexProfiles[team].src === imageUrl
+
+  React.useEffect(() => {
+    void controls.start((_, {rotate = 0}) => {
+      const target =
+        typeof rotate === 'number'
+          ? state === 'initial'
+            ? rotate - 360
+            : rotate + 360
+          : 360
+
+      return {
+        rotate: [rotate, target],
+        transition: {
+          duration: durations[state],
+          repeat: Infinity,
+          ease: 'linear',
+        },
+      }
+    })
+  }, [state, controls])
+
+  return (
+    <Link
+      to={user ? '/me' : hasActiveMagicLink ? '/signup' : '/login'}
+      aria-label={
+        user ? 'My Account' : hasActiveMagicLink ? 'Finish signing up' : 'Login'
+      }
+      className={clsx(
+        'inline-flex items-center justify-center ml-4 w-14 h-14 rounded-full focus:outline-none',
+      )}
+      ref={ref}
+    >
+      <motion.div className="absolute" animate={controls}>
+        <TeamCircle size={56} team={team} />
+      </motion.div>
+      <img
+        className={clsx('inline w-10 h-10 rounded-full select-none', {
+          'object-cover bg-inverse': !isMascotAvatar,
+          'object-contain': isMascotAvatar,
```

Comment:
I've changed these so that Alex looks awesome! For normal photos, we want them to "cover" the image circle. Alex on the other hand is an illustration with a transparent background.